### PR TITLE
Warning about SQLite driver

### LINKED ISSUE #848: SQLite can't be used with the influxd server (OSS or Cloud)
Because InfluxDB doesn't have access to the filesystem, the `sqlite` connector in `sql.from()` and `sql.to()` won't work in InfluxDB OSS or Cloud. It will work in the REPL however. We need to add a note to the docs explaining this.

##### Relevant URLs
- https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/sql/from/
- https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/sql/to/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #848

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/reference/flux/stdlib/sql/to.md [hunk 1]
```diff
@@ -100,8 +100,8 @@ sql.to(
 {{% warn %}}
 **InfluxDB OSS** and **InfluxDB Cloud** do not have direct access to the local filesystem
 and can not write to SQLite data sources.
-Use the [Flux REPL](/v2.0/reference/cli/influx/repl/) to write a SQLite data source
-on your local filesystem.
+Use the [Flux REPL](/v2.0/reference/cli/influx/repl/) to write to an SQLite data
+source on your local filesystem.
```
Comment (rawkode) [inline content/v2.0/reference/flux/stdlib/sql/to.md]: I think this was correct the first time?

File: content/v2.0/query-data/flux/sql.md [hunk 2]
```diff
@@ -69,6 +69,10 @@ sql.from(
 
 {{% code-tab-content %}}
 ```js
+// NOTE: InfluxDB OSS and InfluxDB Cloud do not have access to
+// the local filesystem and can not query SQLite data sources.
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/sql.md]: cannot?

File: content/v2.0/reference/flux/stdlib/sql/to.md [hunk 3]
```diff
@@ -96,6 +96,14 @@ sql.to(
 ```
 
 ### Write data to an SQLite database
+
+{{% warn %}}
+**InfluxDB OSS** and **InfluxDB Cloud** do not have direct access to the local filesystem
+and can not write to SQLite data sources.
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/sql/to.md]: same

File: content/v2.0/reference/flux/stdlib/sql/from.md [hunk 4]
```diff
@@ -85,6 +87,14 @@ sql.from(
 ```
 
 ### Query an SQLite database
+
+{{% warn %}}
+**InfluxDB OSS** and **InfluxDB Cloud** do not have direct access to the local filesystem
+and can not query SQLite data sources.
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/sql/from.md]: same

**Pull request merged**